web/themes/custom/portfolio_theme/scss/_overrides.scss

```diff
@@ -34,20 +34,40 @@ header.navbar {
   .name.navbar-brand {
     color: #ffffff;
     text-transform: uppercase;
+    font-size: 12px;
+    letter-spacing: 8px;
   }
 }
 
 .navbar-header {
   max-width: 1080px;
 }
 
-#block-main {
-  float:right;
+// Main menu
```

Comment:
@leormsbee  You may already know this, but just in case: // for comments works in scss (SASS), but not in regular CSS. So it's OK here, but don't use this directly in a .css file.